omod/src/main/webapp/admin/maintenance/implementationIdForm.jsp

```diff
@@ -30,7 +30,7 @@
 				<td style="white-space: nowrap">
 					<input type="text" value="${status.value}" name="${status.expression}" maxlength="20" size="8"/>
 					<c:if test="${status.errorMessage != ''}">
-						<span class="error">${status.errorMessage}</span>
+						<span class="error"><c:out value="${status.errorMessage}"/></span>
 					</c:if>
```

Comment:
Do we expect cross site scripting in such cases <input type="text" name="${status.expression}" value="${status.value}" size="40" />